devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -29,6 +29,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "000A, 0019", manufacturer: "Jasco Products", model: "45856", deviceJoinName: "GE ZigBee In-Wall Switch"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "PSM_00.00.00.35TC", deviceJoinName: "Ozom Smart Plug"
 		fingerprint profileId: "0104", inClusters: "0000, 0702, 0003, 0009, 0B04, 0006, 0004, 0005, 0002", outClusters: "0000, 0019, 000A, 0003, 0406", manufacturer: "Develco Products A/S", model: "Smart16ARelay51AU", deviceJoinName: "Aurora Smart Inline Relay"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0702", outClusters: "0003, 0019", manufacturer: " ", model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug"	
```

Comment:
Can you try changing the fingerprint to the following and see if it still joins with the correct DTH?

`fingerprint manufacturer: "", model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug"`

If that doesn't work try:

`fingerprint model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug"`

Reply:
I ask because the logs indicate the device is joining based on the clusters and not the manufacturer/model. That could be a bug in the logging or an issue with the fingerprint.

Reply:
@tpmanley 
As you suggested below FP is working fine.
 fingerprint manufacturer: "", model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug"

We have amended PR accordingly. Please review and merge.